README.md

```diff
@@ -1,2 +1,105 @@
 # unified-user-data-format
+
 Specification for the unified user data format across the various "Tube" apps (NewPipe, FreeTube, Invidious, etc..)
+
+## Intro
+
+* This specification is a very early draft, and prone to regular changes
+* Unless otherwise specified, all fields are assumed to be `Strings`
+
+
+## Top level structure
+
+#### `"version"`
+
+The [semantic version](https://semver.org/) of the specification.
+
+Conditions for bumping major/minor/patch version: TBD
+Potential suffixes: `-draft`, TBD
+
+#### `"subscriptions"`
+
+An array of Subscription objects. See below for details.
+
+#### `"playlists"`
+
+An array of Playlist objects. See below for details.
+
+### Overview
+
+```javascript
+{
+	"version": "X.Y.Z[-draft/-rc]",
+	"subscriptions": Subscriptions[],
+	"playlists": Playlist[],
+}
+```
+
+## Subscription object
+
+#### > `"name"`
+
+The name of the subscribed channel.
+
+#### > `"url"`
+
+The URL used to access the user subscribed channel.
+
+#### > `"thumbnail"`
+
+The URL to the user/channel's thumbnail.
```

Comment:
I feel this should be an optional field, since:

- It's not really necessary for importing subscriptions.
- Can change and 404 in the future.

It's also ambiguous whether this should be a direct link to the image or a proxied URL to the media.

Reply:
resolved in https://github.com/UniversalPipeWrench/unified-user-data-format/pull/2/commits/5a9c3d95466dab7a3b11d05be5159beb545a5621